core/lib/contracts/src/lib.rs

```diff
@@ -393,3 +393,314 @@ impl BaseSystemContracts {
         }
     }
 }
+
+pub static COMMIT_BLOCKS_FUNCTION: Lazy<Function> = Lazy::new(|| {
```

Comment:
Could we read it from file?

Reply:
Not from the file we used to read from as in `contracts` submodule there will be only new contracts. I can move old ABI from here to a separate JSON file but I'm not sure where is a good place for it

Reply:
not yet exists, i hope once we will have it